script.js

```diff
@@ -0,0 +1,306 @@
+const suits = [
+    {
+        suit: '♣',
+        color: 'black'
+    },
+    {
+        suit: '♠',
+        color: 'black'
+    },
+    {
+        suit: '♥',
+        color: 'red'
+    },
+    {
+        suit: '♦',
+        color: 'red'
+    }
+]
+
+const faces = [
+    {
+        face: 'A',
+        value: 1
+    },
+    {
+        face: '2',
+        value: 2
+    },
+    {
+        face: '3',
+        value: 3
+    },
+    {
+        face: '4',
+        value: 4
+    },
+    {
+        face: '5',
+        value: 5
+    },
+    {
+        face: '6',
+        value: 6
+    },
+    {
+        face: '7',
+        value: 7
+    },
+    {
+        face: '8',
+        value: 8
+    },
+    {
+        face: '9',
+        value: 9
+    },
+    {
+        face: '10',
+        value: 10
+    },
+    {
+        face: 'J',
+        value: 11
+    },
+    {
+        face: 'Q',
+        value: 12
+    },
+    {
+        face: 'K',
+        value: 13
+    }
+]
+
+const slider = document.getElementById("difficulty")
+const startBtn = document.getElementById("start")
+let side = 'R';
+let difficulty = 0
+let cards = [];
+let drawnCards = [];
+let matches = 0;
+let yourMatches = 0;
+let yourLosses = 0;
+
+function createDeck() {
+    suits.forEach(s => {
+        faces.forEach(f => {
+            let card = {
+                suit: s.suit,
+                color: s.color,
+                face: f.face,
+                value: f.value
+            }
+            cards.push(card);
+        })
+    });
+}
+
+function drawCard(cards) {
+    let rand = Math.floor(Math.random() * cards.length)
+    let chosenCard =  cards.splice(rand, 1)[0]
+    drawnCards.push(chosenCard);
+    return chosenCard
+}
+
+function startTheClock(difficulty) {
+    let count = cards.length;
+    moveCards(drawCard(cards))
+
+    const timer = setInterval(function() {
+        count--;
+        if (count == 0) {
+            clearInterval(timer);
+            console.log("No more cards!");
+            console.log(`${yourMatches} out of ${matches} matches`)
+            let message = document.querySelector("#message>p")
+            message.innerHTML = `You got<br>${yourMatches}<br>out of<br>${matches}<br>matches`
```

Comment:
Can skip this assignment, we'r enot doing anything with it later so no need to 'keep' it in a variable

```suggestion
            document.querySelector("#message>p").innerHTML = `You got<br>${yourMatches}<br>out of<br>${matches}<br>matches`
```